SparkPost/matchbox
# UX-728 Update lockfiles to v2, Upgrade libby

### What Changed

- Upgrades `package-lock.json` files to `v2`
- Upgrades libby

### How To Test or Verify

- Run libby with `npm run start:libby`
- Verify it runs

### PR Checklist
